### DISCUSSION TITLE
[SDK] Error in type exports in signer.d.ts and wallet.d.ts

### DISCUSSION THREAD

**Discussion by everdimension** [zkSync-Community-Hub/zksync-developers#509] [SDKs] [answered]
### Environment

Testnet

### Acknowledgement

- [ ] I understand that issues are expected with the Go SDK as they are being updated.

### Issue Description

I'm trying to use `zksync-ethers` and I'm getting these errors during build:

<details>
<summary><code>node_modules/zksync-ethers/build/signer.d.ts:5:27 - error TS2307: Cannot find module './typechain/Il2Bridge' or its corresponding type declarations.</code></summary>
<pre><code>
node_modules/zksync-ethers/build/signer.d.ts:5:27 - error TS2307: Cannot find module './typechain/Il2Bridge' or its corresponding type declarations.

5 import { Il2Bridge } from './typechain/Il2Bridge';
                            ~~~~~~~~~~~~~~~~~~~~~~~

node_modules/zksync-ethers/build/signer.d.ts:6:25 - error TS2307: Cannot find module './typechain/IZkSync' or its corresponding type declarations.

6 import { IZkSync } from './typechain/IZkSync';
                          ~~~~~~~~~~~~~~~~~~~~~

node_modules/zksync-ethers/build/signer.d.ts:7:27 - error TS2307: Cannot find module './typechain/Il1Bridge' or its corresponding type declarations.

7 import { Il1Bridge } from './typechain/Il1Bridge';
                            ~~~~~~~~~~~~~~~~~~~~~~~

node_modules/zksync-ethers/build/wallet.d.ts:6:25 - error TS2307: Cannot find module './typechain/IZkSync' or its corresponding type declarations.

6 import { IZkSync } from './typechain/IZkSync';
                          ~~~~~~~~~~~~~~~~~~~~~

node_modules/zksync-ethers/build/wallet.d.ts:7:27 - error TS2307: Cannot find module './typechain/Il1Bridge' or its corresponding type declarations.

7 import { Il1Bridge } from './typechain/Il1Bridge';
                            ~~~~~~~~~~~~~~~~~~~~~~~

node_modules/zksync-ethers/build/wallet.d.ts:8:27 - error TS2307: Cannot find module './typechain/Il2Bridge' or its corresponding type declarations.

8 import { Il2Bridge } from './typechain/Il2Bridge';
                            ~~~~~~~~~~~~~~~~~~~~~~~
</code></pre>
</details>

I'm using `parcel` as the bundler:

```
parcel v2.9.3
zksync-ethers v5.7.0
node v20.10.0
npm v10.2.5
```




### Expected Behavior

No errors are expected

### Code Example

import { EIP712Signer } from 'zksync-ethers';


### Repo Link (Optional)

_No response_

**Comment by bxpana** [comment]
Will check this out and see what's up.

**Comment by dutterbutter** [comment]
@everdimension can you please provide a link to the repo so we may reproduce.

**Comment by danijelTxFusion** [comment, marked as answer]
@everdimension Try to use the `zksync-ethers: 5.7.2` version which is made to work with bundlers.

**Reply by everdimension** [reply]
Thanks! Worked
